### PR TITLE
Added json for team page

### PR DESCRIPTION
**Pull request by shivamsingh-07**
There are no profile links for some people.

### CODE REVIEW

File: team/team.json [hunk 1]
```diff
@@ -0,0 +1,277 @@
+{
```
Comment (rajat2502) [inline team/team.json:1]: Is this file formatted? Using ESlint.

File: team/team.json [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+	],
+	"third": [
+		{
+			"profile": "rajat_sir.png",
+			"name": "Rajat Verma",
+			"designation": "Developer",
+			"links": [
+				{
+					"website": "linkedin",
+					"url": "https://www.linkedin.com/in/rajat250"
```
Comment (rajat2502) [inline team/team.json]: Linkedin link broken!
Check for all the links, make sure they are correct!

File: team/team.json [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+					"url": "https://www.linkedin.com/in/saksham-raghuvanshi-0a1b58170/"
+				},
+				{
+					"website": "github",
+					"url": "https://github.com/sraghuvanshi"
+				}
+			]
+		},
+		{
+			"profile": "prajwal_sir.png",
```
Comment (rajat2502) [inline team/team.json]: Change the name of all the files to `name.png` only.

**Review by rajat2502 — COMMENTED** [review]
@shivamsingh-07 and @Shruti3004 
Make these changes asap.

File: team/team.json (hunk 1 above)
Comment (shivamsingh-07) [reply team/team.json:1]: No sir. I just used prettier plugin in vs code.

**Comment by shivamsingh-07** [comment]
@rajat2502 sir I made all the changes. You can review now.

File: team/team.json [hunk 4]
```diff
@@ -0,0 +1,277 @@
+{
+	"forth": [
+		{
+			"profile": "Pranjali.png",
+			"name": "Pranjali Katiyar",
+			"designation": "Chief Executive Member",
```
Comment (rajat2502) [inline team/team.json]: Change `Chief Executive Member` to `President`

File: team/team.json [hunk 5]
```diff
@@ -0,0 +1,277 @@
...
+				{
+					"website": "github",
+					"url": "https://github.com/pr-jli"
+				}
+			]
+		},
+		{
+			"profile": "prakhar.png",
+			"name": "Prakhar Varshney",
+			"designation": "CTC Member, Head of Club: Technical",
```
Comment (rajat2502) [inline team/team.json]: Remove `Member`, and Write HOC's as we have in the current version, 
image

Comment (rajat2502) [reply team/team.json]: Change this for all HOC's

**Review by rajat2502 — COMMENTED** [review]
@shivamsingh-07 make these 2-3 changes, rest all looks good to me. 
Good Work👍

**Comment by rajat2502** [comment]
Also, change the medium logo, this one has rounded edges.

**Comment by shivamsingh-07** [comment]
> Also, change the medium logo, this one has rounded edges.

Sir I can't find a dark mode supported medium svg. Is that ok ?

**Pull request merged**